src/libs/getEmailKeyboardType/index.native.js

```diff
@@ -0,0 +1,8 @@
+
+/**
+ * Return the email-address keyboard type when running on native platforms.
+ * @return {String | null}
```

Comment:
There is no need to add the | null; we are 100% sure about the return type.

```suggestion
 * @return {String}
```